pymbar/fes.py

```diff
@@ -1565,9 +1567,15 @@ def _get_fes_kde(
         if reference_point == "from-lowest":
             fmin = np.min(f_i)
             f_i = f_i - fmin
+            wheremin = np.argmin(
```

Comment:
Need to find the location that this is zeroed at for the actual computation of the std.